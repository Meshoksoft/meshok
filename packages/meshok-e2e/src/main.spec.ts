import puppeteer from "puppeteer";

describe("Meshok", () => {
	let browser: puppeteer.Browser;
	let page: puppeteer.Page;

	beforeAll(async () => {
		browser = await puppeteer.launch();
		page = await browser.newPage();

		await page.goto("http://localhost:3000");
	});

	afterAll(async () => {
		await browser.close();
	});

	it("works", async () => {
		const appHtml = await page.$eval("#root", root => root.innerHTML);

		expect(appHtml).toMatchSnapshot();
	});
});
