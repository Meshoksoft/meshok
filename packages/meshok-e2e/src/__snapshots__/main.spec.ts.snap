// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meshok works 1`] = `

<ul>
  <li>
    <label>
      Hello World!!!
    </label>
    <ul>
      <li>
        <label>
          Foo
        </label>
        <ul>
          <li>
            <label>
              Hello World!!!
            </label>
          </li>
        </ul>
      </li>
      <li>
        <label>
          Baz
        </label>
      </li>
    </ul>
  </li>
</ul>

`;
